crates/epoch-snark/src/api/ffi/utils.rs

```diff
@@ -46,11 +43,11 @@ impl TryFrom<&EpochBlockFFI> for EpochBlock {
 ///
 /// This WILL read invalid data if you give it a larger `num` argument
 /// than expected. Use with caution.
-pub(super) fn read_slice<C: CanonicalDeserialize>(
+pub unsafe fn read_slice<C: CanonicalDeserialize>(
```

Comment:
I actually liked the fact that unsafe was limited to the specific unsafe parts. Wdyt?

Reply:
No because if a bad slice is passed to it it will fail